src/content/docs/game-design/godot/dungeoncrawler/0-scenesetup/index.mdx

```diff
@@ -0,0 +1,70 @@
+---
+type: tutorial
```

Comment:
The first page should explain what the game is with some screenshots and then we can jump into it.

Reply:
Look at the codeclub resource structure for their projects